crates/red_knot_python_semantic/src/semantic_index/except_handlers.rs

```diff
@@ -0,0 +1,79 @@
+use std::cell::{RefCell, RefMut};
```

Comment:
I would make this a sub module of `builder` if it is only used in `builder.rs`. It allows to keep the structs scoped to `builder` (and not the entire `semantic_index`).